cmake/pika_add_test.cmake

```diff
@@ -64,15 +64,17 @@ function(pika_add_test category name)
     set(args ${args} "--pika:bind=none")
   endif()
 
-  set(args "${${name}_UNPARSED_ARGUMENTS}" ${args})
+  set(args "${${name}_ARGS}" "${${name}_UNPARSED_ARGUMENTS}" ${args})
 
   set(_script_location ${PROJECT_BINARY_DIR})
 
   set(cmd ${_exe})
 
   if(${name}_RUNWRAPPER)
+    set(_preflags_list_ ${MPIEXEC_PREFLAGS})
```

Comment:
Is `MPIEXEC_PREFLAGS` a `find_package(mpi)` thing or should we rather name it `PIKA_TEST_MPIEXEC_PREFLAGS` or something like that?

Reply:
its built into `find_package(mpi)`